sig-apps/annual-report-2021.md

```diff
@@ -0,0 +1,60 @@
+- [KubeCon NA 2019](https://www.youtube.com/watch?v=uqYStFxu578)
+
+# Membership
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?  
+Yes, all the lists are up-to-date.
+
+How do you measure membership? By mailing list members, OWNERs, or something else?  
+Reviewers and approvers are measured through repository OWNERS file. Membership is measured through mailing list and zoom calls.
+
+How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?  
+We periodically remove inactive reviewers and approvers, invite new contributors we have seen active in to join the reviewers. The bar to be accepted as an approver is significantly higher since we need to ensure the stability of the core controllers and the entire project.
+
+Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?  
+Recently we reviewed and unified the sig-apps-approvers alias which holds the approvers, similarly sig-apps-reviewers was reviewed for the reviewers ranks.
+
+What programs do you participate in for new contributors?  
+ - KubeCon updates.
+ - Individual/1-1 mentoring.  
+
+Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?  
+The group has contributors from multiple companies/affiliations.
+[14 companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Apps&var-repo_name=kubernetes%2Fkubernetes&var-companies=All) contributed code in the last year.
+
+
+
+# Current initiatives and project health
+
+What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?  
+- [CronJob to GA](https://github.com/kubernetes/enhancements/issues/19)
+- [PDB to GA](https://github.com/kubernetes/enhancements/issues/85)
+
+Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
+What areas and/or subprojects does the group need the most help with?
+ - [New CronJob controller as part of work from promoting CronJobs to GA (alpha)](https://github.com/kubernetes/enhancements/issues/19)
+ - [MaxSurge for DaemonSet (alpha)](https://github.com/kubernetes/enhancements/issues/1591)   
+
+What's the average open days of a PR and Issue in your group? / what metrics does your group care about and/or measure?  
+The average time to merge the PR is around [~5 days](https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?orgId=1&from=1577865600000&to=1609488000000&var-period=y&var-repogroup_name=SIG%20Apps&var-apichange=All&var-size_name=All&var-kind_name=All)
```

Comment:
I'm failing to see this value in the shared link. But I also see that the median says 2.8h, which doesn't sound accurate.
How are these calculated?

Reply:
@alculquicondor you're right, that is off -
Looking at the SIG Apps repo group with the 7 day Moving Average and adding up the open -> LGTM, LGTM -> Approve -> Approve to merge is about 6.9 days.

https://k8s.devstats.cncf.io/d/44/pr-time-to-approve-and-merge?viewPanel=8&orgId=1&from=1577865600000&to=1609401600000&var-period=d7&var-repogroup_name=SIG%20Apps&var-repo_name=kubernetes%2Fkubernetes&var-apichange=All&var-size_name=All&var-kind_name=All